mobile/Yote/package.json

```diff
@@ -11,8 +11,8 @@
     "lodash": "^4.13.1",
     "moment": "^2.13.0",
     "prop-types": "^15.6.0",
-    "react": "^16.0.0",
-    "react-native": "^0.50.3",
+    "react": "^16.3.2",
```

Comment:
did you change the package dependencies on purpose? if so, why?

Reply:
That was unintentional. I was making a mess of things trying to get the mobile part running, I missed that when I was cleaning it up.